src/pages/en/reference/integrations-reference.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro Integration API
+---
+
+## Usage
+
+Astro integrations are always added through the `integrations` property in your  `astro.config.js` file. 
+
```

Comment:
Did you specifically mean `astro.config.js` or do you want `astro.config.mjs` to be consistent with what's in the Integrations Guide page?

(I know @Princesseuh has a docs issue requesting documentation re: this file could be .js, .ts., .mjs etc. So, that should also be on our radar! But, for the sake of consistency between these two documents, we maybe should just pick one/stick with .mjs for now until we document this?)

Reply:
OK! We've always used `.mjs` in other places in just the generic guides info, and that is the default config file (right now!) in our starter templates.

I would say using `.mjs` is probably going to be least disruptive right now (because it might be scattered throughout), and then combined with the request to document how it really could be a few different things, we can delay and address at that time?

Reply:
Update: Oh, the rest of the site does `mjs`! Updating it to that instead.